keylime.conf

```diff
@@ -278,6 +278,17 @@ measured_boot_policy_name = accept-all
 # The default value for this config item is the empty string.
 # measured_boot_imports = keylime.elchecking
 
+# Severity labels for revocation events strictly ordered from least severity to
+# highest severtiy.
+severity_labels = ["info", "notice", "warning", "err", "crit", "alert", "emerg"]
```

Comment:
crit -> critical , err -> error , emerg -> emergency ?

Reply:
Makes sense I changed that.